content/packit-as-a-service.md

```diff
@@ -65,3 +65,15 @@ pull request in COPR repository and reports back COPR build status to the pull r
 An example of Packit Service pull request:
 
 ![Packit service pull request](/packit-service-pr.png)
+
+## FAQ
```

Comment:
FAQ means frequently asked questions and you seem to be not asking any :P

Reply:
Idea for a next question: "How can I reach you out?"

Reply:
How can I reach out to you? :) or How can I contact you?

Reply:
Engrish iz hart.